lib/BC/Util.cpp

```diff
@@ -239,9 +239,14 @@ FindVarInFunction(llvm::Function *function, std::string_view name_,
   return {nullptr, nullptr};
 }
 
+bool IsCompatibleStateFunction(llvm::Function *function) {
```

Comment:
`State` is already pretty overloaded word. Maybe `HasRemillArgPrefix`/`LiftableInto`?